src/Card.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace PodPoint\Payments;
+
+class Card
+{
+    /**
+     * The uid or unique identifier for a payment method.
```

Comment:
`payment method` is a Stripe term - would all card

Reply:
I thought the terminology was right there but yeah could be misleading I'm going to update that to card